homeassistant/components/august/lock.py

```diff
@@ -37,11 +37,16 @@ async def async_setup_entry(
     data: AugustData = hass.data[DOMAIN][config_entry.entry_id]
     async_add_entities(AugustLock(data, lock) for lock in data.locks)
 
+class LockEntityFeature(IntFlag):
+    """Supported features of the lock entity."""
+
+    OPEN = 1
 
 class AugustLock(AugustEntityMixin, RestoreEntity, LockEntity):
     """Representation of an August lock."""
 
     _attr_name = None
+    _attr_supported_features: LockEntityFeature = LockEntityFeature(0)
```

Comment:
```suggestion
    _attr_supported_features = LockEntityFeature.OPEN
```
is this really true for every August lock? As in, some people might have doors that don't have a latch. You probably need to check if this is enabled for this lock in your library first